build-system/tasks/runtime-test.js

```diff
@@ -529,15 +528,15 @@ async function runTests() {
   // Run fake-server to test XHR responses.
   process.env.AMP_TEST = 'true';
   const server = gulp.src(process.cwd(), {base: '.'}).pipe(webserver({
-    port: 31862,
+    port: 8081,
```

Comment:
Future task: Expose the port number via a Karma variable (accessible by tests) instead of hard-coding this value.

Reply:
I started working on this exact thing for this PR, but there were too many moving parts for it to be worth it now. Will create an issue for this